packages/react/src/Hero/Hero.tsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import clsx from 'clsx'
+import { ForwardedRef, forwardRef, HTMLAttributes, PropsWithChildren } from 'react'
+
+export interface HeroProps extends HTMLAttributes<HTMLHeadingElement> {
+  /**
+   * De kleur van de titel
+   * Gebruik deze property om de titel in tegenovergestelde kleur te tonen.
+   */
+  inverseColor?: boolean
+}
+
+export const Hero = forwardRef(
+  (
+    { children, className, inverseColor, ...restProps }: PropsWithChildren<HeroProps>,
```

Comment:
We should replace rendering the `children` with a `text` string prop. We only want text in a hero, right? Also, e.g. the `inverseColor` prop may have no effect on child components.

I had already added a story to Jira to do this for `Heading` and `Button`.

Reply:
I saw the story, I'm not quite sure about that though, for a couple of reasons:

- a bit arbitrary, but as a developer, I prefer writing `<Button>Label</Button>` vs `<Button label="label" />
- if we only allow text, our users aren't able to customize the contents of these containers at all. A button with a loading state for example, which we don't have now, would not be able to be built from our components by a user. And neither would, and that's the most important reason imo, a composition of components we're currently not aware of, but which is necessary in a users project.

Reply:
I understand your second point. But I don’t think we can carry the responsibility of preparing our component with many unknown compositions.

E.g. if people need a spinner, we should add it to the component as a feature, and make sure it works well with all existing features.

Let’s not change it now, add it to that story, discuss more later.